pkg/lokomotive/cluster.go

```diff
@@ -20,12 +20,25 @@ import (
 	"os"
 	"strings"
 	"text/tabwriter"
+	"time"
 
 	v1 "k8s.io/api/core/v1"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 )
 
+const (
+	// Max number of retries when waiting for cluster to become available.
```

Comment:
I think this is not a number anymore, this comment should be changed. What do you think?

Reply:
Yes, thanks.